src/workerd/api/sql.c++

```diff
@@ -197,16 +197,20 @@ SqlStorage::Statement::Statement(SqliteDatabase::Statement&& statement)
 kj::Array<const SqliteDatabase::Query::ValuePtr> SqlStorage::Cursor::mapBindings(
     kj::ArrayPtr<BindingValue> values) {
   return KJ_MAP(value, values) -> SqliteDatabase::Query::ValuePtr {
-    KJ_SWITCH_ONEOF(value) {
-      KJ_CASE_ONEOF(data, kj::Array<const byte>) {
-        return data.asPtr();
-      }
-      KJ_CASE_ONEOF(text, kj::String) {
-        return text.asPtr();
-      }
-      KJ_CASE_ONEOF(d, double) {
-        return d;
+    KJ_IF_MAYBE(non_null_value, value) {
+      KJ_SWITCH_ONEOF(*non_null_value) {
```

Comment:
Nitpick: We don't use underscores in identifier names.

But also, we sort of have a convention that when the second argument to KJ_IF_MAYBE is a plain identifier, then the first parameter should be a single-letter abbreviation of it... otherwise it gets too verbose prefixing everything with `nonNull` or whatever.

```suggestion
    KJ_IF_MAYBE(v, value) {
      KJ_SWITCH_ONEOF(*v) {
```